webpack/scenes/ContentViews/Details/Versions/ContentViewVersionContent.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+
+const ContentViewVersionContent = ({ cvId, versionId, cvVersion }) => {
+  const {
+    deb_count: debCount,
+    docker_manifest_count: dockerManifestCount,
+    docker_tag_count: dockerTagCount,
+    file_count: fileCount,
+    module_stream_count: moduleStreamCount,
+  } = cvVersion;
+
+  return (
+    <React.Fragment>
+      {(moduleStreamCount > 0 &&
+      <React.Fragment id="module_stream_count"><a href={urlBuilder(`content_views/${cvId}/versions/${versionId}/module_streams`, '')}>{`${moduleStreamCount} Module Streams`}</a><br /></React.Fragment>) ||
+        ''
+      }
+      {(debCount > 0 &&
+      <React.Fragment><a href={urlBuilder(`content_views/${cvId}/versions/${versionId}/deb`, '')}>{`${debCount} Deb Packages`}</a><br /></React.Fragment>) ||
+        ''
+      }
+      {(dockerManifestCount > 0 && dockerTagCount > 0 &&
+      <React.Fragment>
```

Comment:
```suggestion
      <>
```
remember we can use the `<>..</>` syntax :)